K8s/deployment.yaml

```diff
@@ -0,0 +1,60 @@
+# deployment.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+ name: module-14
+spec:
+ selector:
+  matchLabels:
+    app: module-14
+ replicas: 1
+ template:
+  metadata:
+    labels:
+      app: module-14
+  spec:
+    containers:
+    - name: todo-app
+      image: todo-app:prod
+      imagePullPolicy: Never
+      ports:
+      - containerPort: 80
+      env:
+      - name: FLASK_APP
+        value: "todo_app/app"
+      - name: FLASK_ENV
+        value: "development"
```

Comment:
This doesn't quite match up with using your production container, but I guess we're falling somewhere between dev & prod for this MiniKube set up!